specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-02-09/Quotas.json

```diff
@@ -0,0 +1,303 @@
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK"
+                    }				
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Compute.Admin/locations/{location}/quotas": {
+            "get": {
+                "x-ms-examples": {
+                    "Get a list of existing Compute quotas.": {
+                        "$ref": "examples/Quotas/List.json"
+                    }
+                },
+                "tags": [
+                    "Quotas"
+                ],
+                "summary": "Lists all Compute quotas.",
+                "description": "Get a list of existing Compute quotas.",
+                "operationId": "Quotas_List",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "Quotas.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "Quotas.json#/definitions/QuotaList"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Quota": {
+            "description": "Holds Compute quota information used to control resource allocation.",
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "description": "Compute Quota properties.",
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/QuotaProperties"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "../2015-12-01-preview/Compute.json#/definitions/Resource"
+                }
+            ]
+        },
+        "QuotaProperties": {
+            "description": "Properties for a Compute Quota",
+            "type": "object",
+            "properties": {
+                "availabilitySetCount": {
+                    "description": "Maximum number of availability sets allowed.",
+                    "format": "int32",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "coresLimit": {
```

Comment:
Folks, what happens if I have two quotas in the plan? Say planA has limit of 4 and planB has a limit of 8. Will subscription have a limit of 12? If yes, then the behavior is correct, but name is somewhat misleading.